packages/rspeedy/lynx-bundle-rslib-config/src/externalBundleRslibConfig.ts

```diff
@@ -0,0 +1,261 @@
+// Copyright 2025 The Lynx Authors. All rights reserved.
+// Licensed under the Apache License Version 2.0 that can be found in the
+// LICENSE file in the root directory of this source tree.
+
+import type { LibConfig, RslibConfig, Rspack, rsbuild } from '@rslib/core'
+
+import { RuntimeWrapperWebpackPlugin as BackgroundRuntimeWrapperWebpackPlugin } from '@lynx-js/runtime-wrapper-webpack-plugin'
+
+import { ExternalBundleWebpackPlugin } from './webpack/ExternalBundleWebpackPlugin.js'
+import { MainThreadRuntimeWrapperWebpackPlugin } from './webpack/MainThreadRuntimeWrapperWebpackPlugin.js'
+
+/**
+ * The Layer name of background and main-thread.
+ *
+ * @public
+ */
+export const LAYERS = {
+  BACKGROUND: 'background',
+  MAIN_THREAD: 'main-thread',
+} as const
+
+/**
+ * The default lib config{@link LibConfig} for external bundle.
+ *
+ * @public
+ */
+export const defaultExternalBundleLibConfig: LibConfig = {
+  format: 'cjs',
+  syntax: 'es2015',
+  autoExtension: false,
+  autoExternal: {
+    dependencies: false,
```

Comment:
How about `peerDependencies`? I think maybe they should also not be externalized.